Gemfile

```diff
@@ -19,7 +19,7 @@ end
 group :build do
   gem 'librarian-repo', :git => 'https://github.com/msutter/librarian-repo.git'
   gem 'cim'
-  gem 'mof'
+  gem 'mof', :git => 'https://github.com/Iristyle/mof.git'
```

Comment:
Let's use the puppetlabs fork instead, which is currently up to date with my repo:

https://github.com/puppetlabs/mof